src/Lean/Data/Lsp/Basic.lean

```diff
@@ -78,6 +78,18 @@ structure Command where
   arguments? : Option (Array Json) := none
   deriving ToJson, FromJson
 
+/-- A snippet string is a template which allows to insert text
+and to control the editor cursor when insertion happens.
+
+A snippet can define tab stops and placeholders with `$1`, `$2`
+and `${3:foo}`. `$0` defines the final tab stop, it defaults to
+the end of the snippet. Variables are defined with `$name` and
```

Comment:
What about "`$0` denotes the final position of the cursor after the placeholders have been filled out, which defaults to the end of the snippet if not provided"?